package.json

```diff
@@ -23,16 +23,11 @@
     "url": "https://github.com/getsentry/sentry-fullstory/issues"
   },
   "homepage": "https://github.com/getsentry/sentry-fullstory#readme",
-  "peerDependencies": {
-    "@fullstory/browser": ">=1.0.0",
-    "@sentry/browser": ">=4.0.0"
-  },
+  "peerDependencies": {},
```

Comment:
What do you think about adding a Sentry peer dep of `>=4 and <=7`? This way if a new major version of the Sentry SDKs breaks the integrations API people won't get this integration

We can peerdep `@sentry/core`, since that should be required by both react native and browser (and all other possible packages).